website/src/pages/users.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+import users from './../../data/users.json';
+import Layout from "@theme/Layout";
+import Container from "../components/Container"
+import Showcase from "../../core/Showcase"
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+export default (props) => {
+  const context = useDocusaurusContext();
+  const siteConfig = context.siteConfig;
+
+  const editUrl = `${siteConfig.customFields.repoUrl}/edit/master/website/data/users.json`;
+
+  const membersToShowcase = users.filter(
+    user => user.isMember,
+  );
+
+  membersToShowcase.sort((a, b) => a.caption.localeCompare(b.caption))
+
+  const othersToShowcase = users.filter(user => !user.isMember);
+
+  return (
+    <Layout>
+      <Container>
+        <div className="showcaseSection">
+          <div className="prose">
+            <h1>Who is Using FDC3?</h1>
+            <p>FDC3 has several industry-leading <a href="https://www.finos.org/become-a-member">member participants.</a></p>
+          </div>
+          <Showcase users={membersToShowcase} />
+          {/* <div className="logos">{showcase}</div> */}
```

Comment:
If this line isn't going to be used, should it be removed?

Reply:
yes, ideally